lblod/frontend-reglementaire-bijlage
# GN-4820: Bump editor and enable copy/pasting variable nodes correctly

### Overview

GN-4820: Bump editor and enable copy/pasting variable nodes correctly

##### connected issues and PRs:

* https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/435
* https://github.com/lblod/ember-rdfa-editor/pull/1195


### Setup

1. Checkout
2. `npm i` 

### How to test/reproduce

See if copy/pasting variable works


https://github.com/lblod/frontend-reglementaire-bijlage/assets/769698/373321ed-5a64-4867-943b-e9b7a913beb9




### Checks PR readiness
